vizro-ai/hatch.toml

```diff
@@ -37,6 +37,7 @@ installer = "uv"
 VIZRO_AI_LOG_LEVEL = "DEBUG"
 
 [envs.default.scripts]
+copy-vivivo = "python tools/copy_visual_vocabulary.py"
```

Comment:
I would actually just remove this script altogether in favour of:
* `copy-vivivo = "cp visual_vocabulary.json path/to/destination`
* even better: `hatch run examples:gen-vivivo` should do the file copying too, so you'd just define the script there as `["python generate_vivivo_json.py", "cp ..."]`. This way there's no need to run two hatch commands to do the whole generate + copy process.

Maybe also change `cp` to `mv` and then there's no need to `.gitignore` it also?